src/collective/blog/indexers/post.py

```diff
@@ -15,12 +15,7 @@ def post_authors(obj: Post):
     """Returns the authors of a blog post."""
     authors = []
     uids = obj.creators
-    for uid in uids:
-        searchResult = api.content.find(portal_type="Author", UID=uid)
-        if len(searchResult) > 0:
-            author = searchResult[0].getObject()
-
-            if author is not None:
-                authors.append(author.title)
-
+    brains = api.content.find(portal_type="Author", UID=uids)
+    for brain in brains:
+        authors.append(brain.title)
```

Comment:
The catalog metadata has Title capitalized:

```suggestion
        authors.append(brain.Title)
```